### DISCUSSION TITLE
`stabilityDays` combined with `internalChecksFilter: "strict"` not always working as expected

### DISCUSSION THREAD

**Discussion by henninghall** [renovatebot/renovate#20437] [Suggest an Idea]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the bug

The [documentation](https://docs.renovatebot.com/configuration-options/#stabilitydays) explains how to make renovate only create pull requests if stability days have passed:
> If you combine stabilityDays=3 and internalChecksFilter="strict" then Renovate will hold back from creating branches until 3 or more days have elapsed since the version was released. 

## Reproducing the bug

1. Use `internalChecksFilter: "strict"` and  `stabilityDays: 100`.
2. Add a dependency that has had a new release the within the last 100 days. 
3. Watch the dependency end up in "Pending" state on the dependecy dashboard (as expected). 
4. Force create a PR for this pending dependency by checking the checkbox. 
5. Close the PR.
6. Check the checkbox under "Ignore or Blocked"
<img width="746" alt="image" src="https://user-images.githubusercontent.com/1652607/219101187-9b84445f-23b4-4261-b533-6272dd300e33.png">

## Expected result 
The dependency will end up in pending state again.

## Actual result
There will be a new PR created even if stability days is not passing. There seems to be no way to get the dependency back in pending state again. 

<img width="862" alt="image" src="https://user-images.githubusercontent.com/1652607/219094295-b465f9e8-1663-48fd-9785-fa0aa585d404.png">

So let's say a PR was force created and then closed for some reason (for instance by accident or because it didn't pass some QA checks), there is no way back to pending state and this dependency will be forever ignored or always created (`internalChecksFilter:"strict"` is ignored).

I've tried manually deleting the created branch with the same result. 

## Reproduction repo
https://github.com/henninghall/renovate-bug-repro

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment]
Converted to discussion as this is working as intended.

Ticking a checkbox for a PR in the dashboard will always result in it being opened. There's no easy way for us to track "state" and have it cycle through states. If you tick it, it opens

**Reply by henninghall** [reply]
Thanks for the quick reply! 

Do you know any workaround for it, like reinstalling the plugin perhaps, could that work? 
Or is it actually impossible to ever get the deps back in pending state again once a PR has been opened?

**Reply by rarkins** [reply]
Rename the existing closed PR (e.g. add any character to the existing title). Then it won't show up as blocking the replacement

**Reply by henninghall** [reply]
Thanks a lot, it worked!
